src/about-us/how-to-raise-issues-and-share-feedback.md

```diff
@@ -0,0 +1,118 @@
+
+Your Head of is an additional source of help and advice for both you and your line manager. If you have any concerns relating to your line manager, you should raise this with your Head of initially.
+
+##### Delivery Leads
+
+Delivery Leads support dxw's client project teams. They help project teams work as productively as possible and will provide support in dealing with any issues that might be getting in the way of that. You should approach your Delivery Lead about issues relating to project team work and client relationships.
+
+##### People team
+
+The People team is there to help with issues relating to your wellbeing, career progression or pay where these can't be resolved initially with support from your line manager, Head of and/or Delivery Lead. As well as any other HR-related issues. They can provide advice and guidance, and hands on support with finding appropriate solutions where needed.
+
+The People team can also be contacted directly about anything you consider to be sensitive or confidential and would prefer not to raise with others.
+
+##### Operational Leadership Team
+
+The Operational Leadership Team is responsible for overseeing the day to day running of the company to implement the business plan. You can contact them about any operational, staffing or client issues that have the potential to improve or block the delivery of our business plan objectives.
+
+The members have a (virtual) open door policy and are happy to be contacted via Slack or email. They represent teams from across the organisation, so the chances are you will already know one of them.
+
+##### Directors
+
+The Directors operate at a more strategic level and have overall responsibility for the leadership of the company and its performance, with a particular focus on its commercial success, external profile and reputation, and staff engagement. They manage any major issues and risks. The Operational Leadership Team will escalate issues to the Directors when they need their input or support to make decisions or act on something.
+
+You can ask questions of the Directors yourself, or via others, at dxw monthly connect meetings, dxw quarterly performance update meetings and Finance clinics. All of the Directors have a (virtual) open door policy and are happy to be contacted via Slack or email if you think there is something they should be made aware of directly or you'd like to discuss with them.
+
+This is where there is the most likely potential overlap with the ‘owner feedback' raised via the Trustees. There are no hard and fast right answers here. You can raise things with both the Directors and Trustees if that makes sense. If you are uncomfortable going to Directors about a particular issue, approaching Trustees might be a good alternative.
+
+#### Policies
```

Comment:
suggestion: as above, this could potentially be an h3 as well

Reply:
Per Slack, I've confirmed the current nesting is correct with Wendy